docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
 
+* Table of Contents
+{:toc}
+
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
It may be a good idea to have bulleted points in this draft Acknowledgements section, so that no one gets left out at the very end.